index-html/index.html

```diff
@@ -0,0 +1,213 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/162dfb3961.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="CSS/style.css" />
+    <link rel="stylesheet" href="CSS/general.css" />
+    <title>Capstone Project1</title>
+  </head>
+  <body>
+    <header class="desktop-nav-header">
+      <div class="top-nav">
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#" class="top-nav-link">English </a>
+        <a href="#" class="top-nav-link">myPage </a>
+        <a href="#" class="top-nav-link last">logout </a>
+      </div>
+      <nav class="desktop">
+        <div class="desktop-logo">
+          <a href="index.html" class="desk-logo">
+            <img src="./images/about-logo.png" alt="logo" />
+          </a>
+        </div>
+        <div class="desktop-nav">
+          <ul class="desktop-nav-lists">
+            <li class="nav-list">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#program">Program</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#speakers">Speakers</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#Sponsors">Sponsor</a>
+            </li>
+            <li class="nav-list">
+              <a href="">
+                <button class="nav-btn btn">CC Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!-- Hero section -->
+    <section class="hero" id="home">
+      <div class="nav-toggle">
+        <div class="burger bar1"></div>
+        <div class="burger bar2"></div>
+        <div class="burger bar3"></div>
+      </div>
+      <nav class="mobile-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#program" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">Speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">CC Campaign</button>
+        </a>
+      </nav>
+      <div class="hero-body">
+        <div class="hero-image">
+          <img class="hero-img" src="./images/title.png" alt="" />
+        </div>
```

Comment:
- [ ] Kindly consider adding this as a header not an image cause obviously you can see that it main heading of the page and that is for semantic reasons

Reply:
Done.